lib/convert.js

```diff
@@ -28,13 +28,14 @@
         Q = require("q");
 
     // Constants
-    var DEFAULT_IMAGE_QUALITY           = 90,
+    var DEFAULT_IMAGE_QUALITY           = "90",
         BACKGROUND_COLOR_FOR_FLATTENING = "#fff";
 
     function _getConvertArgumentsForSettings(settings, binaryPaths) {
         var args    = [],
             format  = settings.format,
-            quality = settings.quality ? String(settings.quality) : null,
+            intQuality = Number.parseInt(settings.quality),
```

Comment:
do we want pareInt here since it will do a floor if there are any decimals. Do you know if decimal values are allowed to passed as quality?